packages/e2e/cypress/e2e/projectPermission.cy.ts

```diff
@@ -6,15 +6,16 @@ describe('Login', () => {
 
         cy.visit('/');
         cy.contains('Settings');
-        cy.get('.bp4-non-ideal-state').should('not.exist');
+        cy.contains('Explore');
+        cy.contains('Welcome, David');
```

Comment:
This is more reliable than `cy.get('.bp4-non-ideal-state').should('not.exist');`